jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/Vessel.java

```diff
@@ -20,10 +23,38 @@
 @AllArgsConstructor
 @Setter(AccessLevel.PRIVATE)
 @Entity
-@Table(name = "")
+@Table(name = "vessel")
 public class Vessel {
   @Id
   @GeneratedValue
   @Column(name = "id", nullable = false)
   private UUID id;
+
+  @Column(name = "vessel_imo_number", length = 7, unique = true)
+  private String imoNumber;
```

Comment:
This is a `unique if valid_until is null` case, so I am not sure that `unique = true` is correct strictly speaking.

Reply:
defined in the IM as
    vessel_imo_number varchar(7) NULL UNIQUE,